app/controllers/sessions_controller.rb

```diff
@@ -1,6 +1,6 @@
 class SessionsController < Clearance::SessionsController
   def create
-    @user = OauthUser.new.find_or_create_from_oauth_hash(auth_hash)
+    @user = OauthUser.new.find_or_create_from_oauth_hash(auth_hash.provider, auth_hash.info)
```

Comment:
http://www.rubydoc.info/github/troessner/reek/Reek/Smells/FeatureEnvy

in this case, the `find_or_create_from_oauth_hash` was accessing `auth_hash` twice, one for each of these two arguments